reference/OpenAPI/validation.md

```diff
@@ -37,6 +37,38 @@ Possible values for the validation properties are:
 
 The default value is `false`.
 
+You can also set `x-flat-validate` more specifically for certain paths and methods, for example:
+
+```yaml
+swagger: "2.0"
+x-flat-validate:
+  request: report-only
+paths:
+  # Reports invalid requests (top-level fallback).
+  # Does not validate responses (default).
+  /relaxed:
+
+  # Validates all incoming requests and responses for GET.
+  # Reports invalid responses for POST.
```

Comment:
`request: true` gilt doch auch für `post`, richtig? Also

Validates incoming requests and reports invalid responses for POST.

Reply:
Ja, der erste Satz ist missverständlich formuliert. Es sollte bedeuten: _Validates **all** incoming requests._ und _Validates all responses for GET._

Reply:
> Validates incoming requests and reports invalid responses for POST.

Bei Deinem Vorschlag fehlt die Response-Validierung für GET.

Reply:
Mein Vorschlag bezieht sich nur auf auf Zeile, in der POST vorkommt, weil ich annahm, dass sich die erste nur auf GET bezieht. Ich würde es jetzt in drei Teile auftrennen:

Validates all incoming requests.
Vallidates responses for GET.
Reports invalid responses for POST.

Reply:
:+1: